Fix inconsistent test results for webvtt/paring tests

cola119 opened the pull request:
#### 4fd46adb8769322fc06e7d39b8d9cea72f6d01b6
<pre>
Fix inconsistent test results for webvtt/paring tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=262010">https://bugs.webkit.org/show_bug.cgi?id=262010</a>

Reviewed by Dan Glastonbury.

Even though these tests have succeeded, they are labeled as failures
due to the default setting not displaying the caption. To fix this issue,
we should set the caption mode to `automatic` by the captionDisplayMode option.

* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/align-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/align.html:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/line-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/line.html:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/snapToLines-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/snapToLines.html:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/text-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/text.html:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/vertical-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/vertical.html:

Canonical link: <a href="https://commits.webkit.org/268638@main">https://commits.webkit.org/268638@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/9152885c49d83bcf2e8f0eed9dc244eba0f7f575

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/19505 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/19924 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/20532 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/21395 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/18241 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/23187 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/20066 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/19845 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/19721 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/19746 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/16951 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/22251 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/16933 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/17739 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/24051 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/17994 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/17914 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/22022 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/18521 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/15972 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/17665 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/17574 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/4858 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/22021 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/18349 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk commented on LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/align.html:
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!doctype html><!-- webkit-test-runner [ captionDisplayMode=automatic ] -->
```
Would this not get removed the next time we run an import?

cola119 replied:
This comment is preserved the next time it is imported. `test_importer.py` checks if the existing test has a `webkit-test-runner` comment and inserts it back in the new test file. See https://github.com/WebKit/WebKit/commit/24470f7b90f2685ea50b0e268d212d34f5b46d0c for details.

annevk reviewed (commented):
Thanks, non-reviewer LGTM.

djg reviewed (approved):
Carry review from @annevk

cola119 committed:
Fix inconsistent test results for webvtt/paring tests
https://bugs.webkit.org/show_bug.cgi?id=262010

Reviewed by Dan Glastonbury.

Even though these tests have succeeded, they are labeled as failures
due to the default setting not displaying the caption. To fix this issue,
we should set the caption mode to `automatic` by the captionDisplayMode option.

* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/align-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/align.html:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/line-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/line.html:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/snapToLines-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/snapToLines.html:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/text-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/text.html:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/vertical-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/vertical.html:

Canonical link: https://commits.webkit.org/268638@main
